.travis.yml

```diff
@@ -92,7 +92,7 @@ install:
 # END MRR 
   - cmake -H. -Bbuild
   - cd build
-  - travis_wait 40 make
+  - travis_wait 60 make
```

Comment:
Kind of surprised this is taking as long as it does. One strategy is pull out as much of the C++ headers from the polynomial files as you can. You C can compile 1-2 orders of magnitude faster than C++. It seems a bit silly, but I noted its much more enjoyable to work on things that dont require half an hour to compile.

Reply:
What do you mean? Comile the polynomial files using gcc/icc instead of g++/icpc?

Reply:
No, simply remove as many C++ headers from being included into the kernel files as possible. If its not possible, its not a big deal. Just a suggestion.

Reply:
I will see if I can remove some of them.